MetaData/work/prepareCrabJobs.py

```diff
@@ -247,7 +256,12 @@ def dumpCfg(options):
         position = ProcessedDataset.find("-v")
         processedLabel = ProcessedDataset[:position]
         # print processedLabel
-
+        
+        # apprend extra parameters
+        if options.extraPyCfgParam:
+            #print "options.extraPyCfgParam:",options.extraPyCfgParam
+            replacements["PYCFG_PARAMS"].append(str("%s" % options.extraPyCfgParam))
```

Comment:
this would work only if the specified option is just one. I would be better to use "append" as action for extraPyCfgParam.
 
https://docs.python.org/2/library/optparse.html#standard-option-actions 

the call above would then become `replacements["PYCFG_PARAMS"].extend(map( lambda x: '"%s"' % x, options.extraPyCfgParam  )`